src/index.html

```diff
@@ -20,32 +20,17 @@
   </head>
 
   <body>
-    <div id="container">
+    <div class="container">
       <div id="header">
         <h1>Todo List</h1>
         <div class="iconss"><i class="fas fa-sync"></i></div>
       </div>
-      <div id="inputContainer">
-        <div class="input-line">
-          <div class="in">
-            <input
-              id="taskInput"
-              type="text"
-              placeholder="Add to your list ....."
-              maxlength="50"
-            />
-          </div>
-
-          <div><a href="#"></a><i class="fas fa-level-down-alt"></i></div>
-        </div>
-        <div id="listWrapper">
-          <ul id="taskList"></ul>
-        </div>
-
-        <div id="deleteBtnWrapper">
-          <div class="footer"><a href="#"> Clear all completed</a></div>
-        </div>
+      <div class="items-container">
+        <!-- <div class="title"></div> -->
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.